.github/BOTMETA.yml

```diff
@@ -436,6 +436,10 @@ files:
     keywords: [ ovirt, RHEV, RHV ]
     labels: [ cloud, ovirt ]
     maintainers: $team_ovirt
+    deprecated:
```

Comment:
These keys go into the Modules `DOCUMENTATION`, rather than BOTMETA

Reply:
yeah sorry my mistake already working on it